prisma/schema.prisma

```diff
@@ -45,32 +45,41 @@ model Organization {
 }
 
 model Project {
-  id                Int           @id @default(autoincrement())
-  name              String        @unique
+  id                Int            @id @default(autoincrement())
+  name              String         @unique
+  slug              String?
   field             String
   description       String
   startDate         DateTime?
   endDate           DateTime?
   organizationId    Int?
-  monetaryGoal      Int?
-  organization      Organization? @relation(fields: [organizationId], references: [id])
+  mpEnabled         Boolean?       @default(false)
+  mpInstantCheckout Boolean?       @default(false)
+  mpPublicKey       String?
+  mpAccessToken     String?
```

Comment:
So, the projects will have their own MP account and not the orgs?

Reply:
I will try with that approach, so to avoid holding money in out side. But if it is not possible, we eill just ignore those fields.